src/main/cli/settings.c

```diff
@@ -1050,6 +1050,8 @@ const clivalue_t valueTable[] = {
 #ifdef USE_AIRMODE_LPF
     { "transient_throttle_limit",   VAR_UINT8 | MASTER_VALUE, .config.minmax = { 0, 30 }, PG_PID_PROFILE, offsetof(pidProfile_t, transient_throttle_limit) },
 #endif
+    { "ff_boost",                   VAR_UINT8 | PROFILE_VALUE,  .config.minmaxUnsigned = { 0, 250 }, PG_PID_PROFILE, offsetof(pidProfile_t, ff_boost) },
+    { "ff_boost_hz",                VAR_UINT16 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 200 }, PG_PID_PROFILE, offsetof(pidProfile_t, ff_boost_hz) },
```

Comment:
Parameter names should be `feedforward_` to be consistent with other settings. Users won't know what "ff" means.

Reply:
I don't think ff is likely to be confusing.  
We use P as an abbreviation to Proportional, D for Derivative, and F for feedforward as in PIDF in the CLI, and lots of other abbreviations.   
My personal view is that this parameter should be in the configurator, where long names can be used, don't add to the size of the hex file.  For users who seek to modify this behaviour, some recourse to the wiki is likely to begin to understand what to do with what parameter, there they will see the CLI abbreviation.
So I think we should use ff in place of all reference to feedforward in the CLI.  That saves 8 characters of hex size per usage.  There is another PR for additional feed forward related stuff; each time it adds up.

Reply:
Fixed